src/types/onyx/Domain.ts

```diff
@@ -20,8 +20,14 @@ type Domain = OnyxCommon.OnyxValueWithOfflineFeedback<{
     /** Whether domain validation is pending */
     isValidationPending?: boolean;
 
-    /** Whether validation code is currently loading */
+    /** Errors that occurred when validating the domain */
+    domainValidationError?: OnyxCommon.Errors;
```

Comment:
NAB: Should this be `domainVerificationError` instead? (Though i guess this would involve changing the backend too if so)

Reply:
Just for linguistic consistency, maybe we can do a follow up if we decide to change it

Reply:
we use the term "validation" internally, and "verification" for user facing parts. the ideal here imo would be "validationError", to not repeat "domain" unnecessarily, but I think it's fine. we can change it in the future though